test/e2e/menuPage.cy.js

```diff
@@ -1,8 +1,13 @@
+import {getData} from './commands'
+
 describe('Menu Page', () => {
-  const pageTitle = 'Adapt Version 5';
 
-  it(`should have the title ${pageTitle}`, () => {
+  beforeEach(() => {
```

Comment:
I think we could reduce calls here by just making this a `before` for one run instead of on each test scenario